WordPress/Classes/Services/PostService+Revisions.swift

```diff
@@ -27,9 +27,7 @@ extension PostService {
                                         let revisions = self.syncPostRevisions(from: remoteRevisions ?? [],
                                                                                for: postId.intValue,
                                                                                with: blogId.intValue)
-                                        ContextManager.sharedInstance().save(self.managedObjectContext, completion: {
-                                            success(revisions)
-                                        }, on: .main)
```

Comment:
The `completion` block is called on main queue, where as the `revisions` objects are bound to the `managedObjectContext` queue. In theory, here lays a Core Data concurrency issue, if the `managedObjectContext` is not bound to the main queue.